src/vcpkg/base/downloads.cpp

```diff
@@ -672,107 +132,168 @@ namespace vcpkg
         return true;
     }
 
-    static std::vector<int> curl_bulk_operation(DiagnosticContext& context,
-                                                View<Command> operation_args,
-                                                StringLiteral prefixArgs,
-                                                View<std::string> headers,
-                                                View<std::string> secrets)
+    struct CurlRequestPrivateData
```

Comment:
I think we can avoid this type existing: https://github.com/vicroms/vcpkg-tool/pull/2